From 73db316ef2c935f4a80c39664f0ebbf9e0035c40 Mon Sep 17 00:00:00 2001
From: David Michael <david.michael@coreos.com>
Date: Tue, 16 Aug 2016 13:43:18 -0700
Subject: [PATCH] qemu: include the qemu_fw_cfg kernel module

This adds support for /sys/firmware/qemu_fw_cfg on QEMU guests
started with the -fw_cfg option.
---
 modules.d/90qemu/module-setup.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/90qemu/module-setup.sh b/modules.d/90qemu/module-setup.sh
index 8cae124..9f8495a 100755
--- a/modules.d/90qemu/module-setup.sh
+++ b/modules.d/90qemu/module-setup.sh
@@ -27,5 +27,6 @@ installkernel() {
             ata_piix ata_generic pata_acpi cdrom sr_mod ahci \
             virtio_blk virtio virtio_ring virtio_pci \
             virtio_scsi virtio_console virtio_rng \
-            spapr-vscsi
+            spapr-vscsi \
+            qemu_fw_cfg
 }
